src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -723,17 +724,34 @@ namespace HVACVariableRefrigerantFlow {
                 CoolOABoundary = CurveValue(VRF(VRFCond).CoolBoundaryCurvePtr, InletAirWetBulbC);
                 if (OutdoorDryBulb > CoolOABoundary) {
                     if (VRF(VRFCond).CoolCapFTHi > 0) TotCoolCapTempModFac = CurveValue(VRF(VRFCond).CoolCapFTHi, InletAirWetBulbC, CondInletTemp);
+                }
+            }
+            if (VRF(VRFCond).EIRCoolBoundaryCurvePtr > 0) {
```

Comment:
Split out test for EIR boundary so that capacity and EIR are evaluated individually.